scripts/clear-signed-agreements.py

```diff
@@ -41,13 +47,22 @@ def main(stage, framework_slug, api_token, user):
     )
 
     suppliers = api_client.find_framework_suppliers(framework_slug, agreement_returned=True)['supplierFrameworks']
-    for supplier in suppliers:
+
+    if supplier_ids is not None:
+        supplier_ids = [
+            str(supplier['supplierId']) for supplier in suppliers
```

Comment:
I think they are only used in two places later in this script:

1. sent as an argument to `find_framework_suppliers`
2. in a comparison with the result of a regular expression match when filtering the contents of the bucket

I was assuming both of these can be strings and wanted `supplier_ids` to contain one type, whether they were sent in as options or not. Is there another reason to keep them as `int`?